storage/Volume.cc

```diff
@@ -2441,7 +2441,11 @@ namespace storage
     string Volume::getDmcryptName() const
     {
 	string nm;
-	if( !mp.empty() )
+        if ( mp == "/" )
```

Comment:
And the other thing is yet again a mixture of tabs vs. blanks in indentation: I disabled tabs because of our Ruby style, but the existing code still uses it in some places, thus the visual difference. But it actually looks the same (i.e. it's the same indentation level) outside of GitHub.